editor/store/selectors.js

```diff
@@ -966,7 +920,8 @@ export function isBlockInsertionPointVisible( state ) {
  * @return {boolean} Whether post is being saved.
  */
 export function isSavingPost( state ) {
-	return state.saving.requesting || isSavingMetaBoxes( state );
+	// TODO: filter
```

Comment:
In what way or reason are you suggesting "filter" ?

__Edit:__ I guess so we wouldn't have to pass `forceIsDirty` and `forceIsSaving` as props?

Reply:
Yes, actually I added these comments before adding the props, I think I can remove them right now :)